apps/concierge_site/lib/controllers/auth_controller.ex

```diff
@@ -44,7 +44,9 @@ defmodule ConciergeSite.AuthController do
 
     {:ok, logout_uri} = UeberauthOidcc.initiate_logout_url(auth, logout_params)
 
-    SessionHelper.sign_in(conn, user, %{logout_uri: logout_uri})
+    conn
+    |> put_session("logout_uri", logout_uri)
+    |> SessionHelper.sign_in(user)
```

Comment:
suggestion (non-blocking): `SessionHelper.sign_in` probably doesn't need to accept the `claims` param any longer.

Reply:
That is also a good point, probably not. We did have a React admin section back in the day, but Cora pulled that out a couple of years back.